dockerfiles/README.md

```diff
@@ -13,3 +13,23 @@ docker build -f dockerfiles/header_sync/Dockerfile . -t header_sync:latest
 ```
 docker run -e DATABASE_USER="user" -e DATABASE_PASSWORD="pw" -e DATABASE_HOSTNAME="host" -e DATABASE_PORT="port" -e DATABASE_NAME="name" -e STARTING_BLOCK_NUMBER=0 -e CLIENT_IPCPATH="path" -t header_sync:latest
 ```
+
+## resetHeaderCheckCount
+Dockerfile for resetting the `headers.check_count` to zero in the database for the given header, so that the execute command
+will transform the associated event.s This is useful in case an event log is missing.
+
+### Build
+From project root directory:
+```
+docker build -f dockerfiles/reset_header_check_count/Dockerfile . -t reset_header_check_count:latest
+```
+
+### Run
+```
+docker run -e CLIENT_IPCPATH="https://parity0.mainnet.makerfoundation.com:8545" -e DATABASE_USER="user" -e DATABASE_PASSWORD="password" -e DATABASE_HOSTNAME="host" -e DATABASE_PORT="port" -e DATABASE_NAME="name" -e HEADER_BLOCK_NUMBER=0 -t reset_header_check_count:latest
```

Comment:
maybe want to use a fake example ipc path?

also I've been moving toward removing quotes from these values, bc of weird issues where quotes seem to cause them to parse incorrectly